plugins/scaffolder/src/components/fields/OwnedEntityPicker/OwnedEntityPicker.tsx

```diff
@@ -28,13 +28,22 @@ import useAsync from 'react-use/lib/useAsync';
 
 import { FieldExtensionComponentProps } from '../../../extensions';
 
+/**
+ * The input props that can be specified under `ui:options` for the
+ * `OwnedEntityPicker` field extension.
+ *
+ * @public
+ */
 export interface OwnedEntityPickerUiOptions {
   allowedKinds?: string[];
   defaultKind?: string;
 }
 
 /**
- * Owned Entity Picker
+ * The underling component that is rendered in the form for the `OwnedEntityPicker`
```

Comment:
```suggestion
 * The underlying component that is rendered in the form for the `OwnedEntityPicker`
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>